### DISCUSSION TITLE
liveview image uploader

### DISCUSSION THREAD

**Discussion by mrdotb** [petalframework/petal_components#292] [Ideas]
:wave: Hello

I have been working on a image uploader (liveview only) inspired by the one on reedit. If you are interested I would like to add it to petal_components.

https://github.com/petalframework/petal_components/assets/7212914/d6614967-576d-4bd5-9792-2a45bcdeb432

**Comment by mplatts** [comment]
Yeah that would be cool. I assume it will be tightly coupled to a live view though - sometimes it can be hard to make it into a usable "component" that anyone can plug in.

**Comment by mrdotb** [comment]
It use the components [live_file_input/1](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#live_file_input/1) and [live_img_preview/1](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#live_img_preview/1) under the hood and a bit of JS.

I will make a PR with my try next week.
